### PR TITLE
Add new channels from Chile

### PR DESCRIPTION
**Pull request by sguinetti**
This PR is for add new channels and edit RTVE channels

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -2914,10 +2926,10 @@ Canal22Internacional.mx,Canal 22 Internacional,,,,MX,,,c/US,spa,,FALSE,,,,https:
...
+Canal24Horas.es,Canal 24 Horas,,24,RTVE,ES,,,c/ES,spa;eng,,FALSE,,,,http://www.rtve.es/alacarta/tve/24-horas/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasCanarias.es,Canal 24 Horas Canarias,,24,RTVE,ES,ES-CN,,s/ES-CN,spa;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-canarias/24-h-can/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasCatalunya.es,Canal 24 Horas Catalunya,,24,RTVE,ES,ES-CT,,s/ES-CT,cat;eng,,FALSE,,,,https://www.rtve.es/play/videos/directo/tve-catalunya/24h-cat/,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
+Canal24HorasInternacional.es,Canal 24 Horas Internacional,,24,RTVE,ES,,,r/AMER;r/EUR,spa;eng,,FALSE,,,,https://www.rtve.es/television/tve-internacional,https://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Logo_TVE-24h.svg/512px-Logo_TVE-24h.svg.png
```
Comment (freearhey) [inline data/channels.csv:2932]: I assume this is a typo, because the 4th column should be the name of the network.

File: data/channels.csv [hunk 2]
```diff
@@ -13801,12 +13813,12 @@ KZTVDT1.us,KZTV-DT1,,CBS,,US,US-TX,Corpus Christi,s/US-TX,eng,,FALSE,,,,http://w
...
+La1Canarias.es,La 1 Canarias,,1,RTVE,ES,ES-CN,,s/ES-CN,spa,general,FALSE,,,,http://www.rtve.es/television/tve-canarias/,https://upload.wikimedia.org/wikipedia/commons/thumb/1/19/Logo_TVE-1.svg/512px-Logo_TVE-1.svg.png
+La1Catalunya.es,La 1 Catalunya,,1,RTVE,ES,ES-CT,,s/ES-CT,cat,general,FALSE,,,,https://www.rtve.es/television/catalunya/,https://upload.wikimedia.org/wikipedia/commons/thumb/1/19/Logo_TVE-1.svg/512px-Logo_TVE-1.svg.png
+La2.es,La 2,,2,RTVE,ES,,,c/ES;c/AD,spa,documentary,FALSE,,,,https://www.rtve.es/television/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Logo_TVE-2.svg/512px-Logo_TVE-2.svg.png
+La2Canarias.es,La 2 Canarias,,2,RTVE,ES,ES-CN,,s/ES-CN,spa,documentary,FALSE,,,,http://www.rtve.es/television/tve-canarias/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Logo_TVE-2.svg/512px-Logo_TVE-2.svg.png
+La2Catalunya.es,La 2 Catalunya,,2,RTVE,ES,ES-CT,,s/ES-CT,cat,documentary,FALSE,,,,https://www.rtve.es/television/catalunya/,https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Logo_TVE-2.svg/512px-Logo_TVE-2.svg.png
```
Comment (freearhey) [inline data/channels.csv:13821]: Same here.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 1 above)
Comment (sguinetti) [reply data/channels.csv]: OK. I delete the name for now. I move the name as owner only.

Comment (sguinetti) [reply data/channels.csv]: Wait a minute, would it be okay to set the name of the network as "Canal 24"?

Comment (freearhey) [reply data/channels.csv]: The value of "network" is still incorrect, according to the description on wikipedia it is "Televisión Española" (https://en.wikipedia.org/wiki/24_Horas_(Spanish_TV_channel))

**Review by freearhey — CHANGES_REQUESTED** [review]
> Error: row 37 has the wrong number of columns (data/channels.csv)

https://github.com/iptv-org/database/actions/runs/3568511671/jobs/5997460835

**Comment by sguinetti** [comment]
Sorry again, I will fix in a few moments.

File: data/channels.csv [hunk 3]
```diff
@@ -4394,7 +4409,7 @@ CKYFTV.ca,CKYF-TV,,CTV,,CA,CA-MB,Flin Flon,s/CA-MB,eng,,FALSE,,,,,https://upload
 CKYPTV.ca,CKYP-TV,,CTV,,CA,CA-MB,The Pas,s/CA-MB,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/CTV_logo_2018.svg/512px-CTV_logo_2018.svg.png
 CKYTTV.ca,CKYT-TV,,CTV,,CA,CA-MB,Thompson,s/CA-MB,eng,,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/thumb/1/17/CTV_logo_2018.svg/512px-CTV_logo_2018.svg.png
 CL3Cablevision.ar,CL3 Cablevision,CL3,,,AR,AR-S,Chañar Ladeado,s/AR-S,spa,,FALSE,,,,http://www.cl3cablevision.com.ar/,https://lh3.googleusercontent.com/-Dwon9sRZsSY/Xjs6WmRFAaI/AAAAAAAAvnM/Zgcsd4pznrUNtvK8TghlckQw1YcLWnkdwCK8BGAsYHg/s0/2020-02-05.png
-ClanTVE.es,Clan TVE,,RTVE,,ES,,,c/ES,spa,kids,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/5/5d/Tve_clan.png
+ClanTVE.es,Clan TVE,,Clan,RTVE,ES,,,c/ES,spa,kids,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/5/5d/Tve_clan.png
```
Comment (freearhey) [inline data/channels.csv:4412]:
```suggestion
Clan.es,Clan,,TVE,RTVE,ES,,,c/ES,spa,kids,FALSE,,,,,https://upload.wikimedia.org/wikipedia/commons/5/5d/Tve_clan.png
```

Source: https://en.wikipedia.org/wiki/Clan_(TV_channel)

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 3 above)
Comment (sguinetti) [reply data/channels.csv:4412]: Done

**Review by freearhey — APPROVED** [review]

**Pull request merged**